.travis.yml

```diff
@@ -24,12 +28,20 @@ compiler:
   - clang
   - gcc
 
+git:
+  depth: 5
+
 env:
+  global:
+    - secure: "VezRbHFg6kllV5WG06M3tG3aHJaC3xrMylJ6RCVbL+uz2JeralVCqV7eIk4fVb9cu83Li+weEa0AJj0wkxpIUJ+vUh5F65L6gSWSbgHP7muOSVsmnEc6KvX4n3av/ZGe4geSmsxqh2pd/2xI1h7KioGRhKeqaZIdjVgWgGJW2iQ="
```

Comment:
Are both of these needed? It would be good for there to be a comment re what these tokens are for.